internal/model/label_pair.go

```diff
@@ -0,0 +1,6 @@
+package model
+
+type LabelPair struct {
```

Comment:
this is fine for now, but i feel like we're eventually going to need to split up the k8s package into k8s data structures and k8s client code.